dataprofiler/profilers/profiler_options.py

```diff
@@ -841,7 +841,8 @@ def _validate_helper(self, variable_path='DataLabelerOptions'):
 class TextProfilerOptions(BaseInspectorOptions):
 
     def __init__(self, is_enabled=True, is_case_sensitive=True,
-                 stop_words=None):
+                 stop_words=None, num_most_common_words=None,
+                 num_most_common_chars=None):
```

Comment:
this is a nit picky comment, but is it strange to have words before chars since words are a combination of chars?

Reply:
just want to keep it in the same order as we have in the function, e.g., 
```
self.words = BooleanOption(is_enabled=True)
self.vocab = BooleanOption(is_enabled=True)
```

Reply:
changed the order for all text profiler